provider/service_resources.go

```diff
@@ -84,6 +113,11 @@ func createResourceFireHydrantService(ctx context.Context, d *schema.ResourceDat
 		Labels:      labels,
 	}
 
+	if o, ok := d.GetOk("owner"); ok {
+		os := o.([]interface{})[0].(map[string]interface{})
```

Comment:
I know this is how you have to do this but 😱

Reply:
@bobbytables you've mentioned in one talk somewhere that there is a lot of `interface` going around, no joke!